### PR TITLE
Refactor & more search option

### LINKED ISSUE #7: Is it possible to sort Everything query results by date modified?
How do I sort results by date modified?

I am aware of the dm: functions but the results with this criteria are still sorted by name. I am looking to be able to simply sort the results by most recently modified.

This is easy in the stand-alone Everything app as there are column headers but I can't find how to replicate this in FlowLauncher...

Any ideas?

### LINKED ISSUE #27: Opening Files With Custom Editor Still Not Working When Spaces In Path
hi - i see several updates ostensibly addressing an issue opening files when their paths have spaces - i'm on the current version of the plugin adn the current flow launcher, and i'm still not able to open files with custom editor.

Custom editor path:  "C:\Program Files\IDM Computer Solutions\UltraEdit\uedit64.exe"
Settings:  
image

Error:  
image

Works fine when no spaces in path.  If you look at the error - it looks like it's using the app folder as the working folder and then merging with the actual file path after the first space (the full path of the file i was trying to open was:  D:\Dropbox\Documents\Backups and Maintenance\Automation\Batch Files\Settings Backups - Cleanup.bat)

Can't put my finger on what makes the difference - but i've noticed sometimes it uses the windows directory as the working folder - but gives me basically the same error except instead of starting with flow's appdata folder, the first half of the path in the error message is c:\windows\system32.

also, sometimes ultraedit tries to open a separate file for each space in the path.

### PR DESCRIPTION
**Pull request by taooceros**
close #7 
close #27
Remove old infrastructure dll
Bump plugin version to 2.0.0
https://github.com/jjw24/Droplex/pull/11

### CODE REVIEW

**Comment by taooceros** [comment]
@thunderousleader10 Do you want to take a try? You may need to update everything to the newest version.

Replace all content in %appdata%\FlowLauncher\Plugins\Everything-1.4.9\

[Flow.Launcher.Plugin.Everything.zip](https://github.com/Flow-Launcher/Flow.Launcher.Plugin.Everything/files/6859953/Flow.Launcher.Plugin.Everything.zip)

File: .gitignore [hunk 1]
```diff
@@ -156,3 +156,6 @@ $RECYCLE.BIN/
 .DS_Store
 /.vs/slnx.sqlite
 /.vs/Flow.Launcher.Plugin.Everything/DesignTimeBuild/.dtbcache.v2
+
+# Rider
+/.idea
```
Comment (jjw24) [inline .gitignore]: this is something required? (ok if it is)

Comment (taooceros) [reply .gitignore]: For rider config. Or else there will be some editor config files.

File: Flow.Launcher.Plugin.Everything.csproj [hunk 2]
```diff
@@ -84,20 +68,25 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>Settings.settings</DependentUpon>
     </Compile>
+    <Compile Remove="Output\**" />
```
Comment (jjw24) [inline Flow.Launcher.Plugin.Everything.csproj:71]: think there is a better way to do this right instead of add this to here, how's flow doing it?

**Comment by taooceros** [comment]
I haven't finish the international part for this due to enum selection.

File: Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "Flow.Launcher.Plugin.Everything": {
+      "commandName": "Project"
+    },
+    "Flow Launcher": {
+      "commandName": "Executable",
+      "executablePath": "D:\\Document\\Dotnet\\Flow.Launcher\\Output\\Release\\Flow.Launcher.exe"
```
Comment (jjw24) [inline Properties/launchSettings.json]: does this launchSettings.json file need to be checked in?

**Comment by jjw24** [comment]
> international part

 which international part ?

File: Properties/launchSettings.json (hunk 3 above)
Comment (taooceros) [reply Properties/launchSettings.json]: Oh no, this should be ignored.

File: Flow.Launcher.Plugin.Everything.csproj (hunk 2 above)
Comment (taooceros) [reply Flow.Launcher.Plugin.Everything.csproj:71]: Hmmm I am not sure, will take a look once I get back to home.

Comment (jjw24) [reply Flow.Launcher.Plugin.Everything.csproj:71]: all good no stress, if i have time in front of comp i will do it plus remove the launchSettings.json file

**Comment by taooceros** [comment]
> > international part
> 
>  which international part ?

Translation

**Comment by jjw24** [comment]
> Translation

oh ok. 

I just released Droplex 1.4.0, should be good to switch soon

**Review by jjw24 — APPROVED** [review]

**Comment by ghost** [comment]
> @thunderousleader10 Do you want to take a try? You may need to update everything to the newest version.
> 
> Replace all content in %appdata%\FlowLauncher\Plugins\Everything-1.4.9\
> 
> [Flow.Launcher.Plugin.Everything.zip](https://github.com/Flow-Launcher/Flow.Launcher.Plugin.Everything/files/6859953/Flow.Launcher.Plugin.Everything.zip)

Works perfectly. Thank you for adding!

**Pull request merged**